content/docs/concepts/policies-and-telemetry/istio-metrics.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Istio Metrics
+description: Metrics exported from Istio through Mixer.
+weight: 50
+---
+
+Istio exports metrics through Mixer. They can be found [here](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml)
+under the section with “kind: metric”.
+
+We will describe metrics first and then the labels for each metric.
+
+## Metrics
+
+### Request Count
+
+This is a COUNTER metric incremented for a new request to Istio.
+It is labeled based on source service, source version, destination service, destination version, connection mTLS and response code of the request.
+This is exported by default by prometheus adapter and can be configured to be exported by other mixer adapters.
+
+### Request Duration
+
+This is a DISTRIBUTION metric which measures the duration of the request.
```

Comment:
add: duration of the request (as observed by the server-side proxy).

Reply:
Fixed